Makefile

```diff
@@ -8,7 +8,7 @@ INSTALL_DIR      ?= /usr/local
 vendor:
```

Comment:
Another thing I've also noticed is not having perms to run the binary in ```/usr/local```. Maybe we should just go with ```.``` or make a local dir?